### DISCUSSION TITLE
Evernote conversion issues ...

### DISCUSSION THREAD

**Discussion by CoachKidd** [akosbalasko/yarle#499] [Q&A] [answered]
Hi @akosbalasko ...

A big thank you for providing an escape from Evernote to Obsidian.  I'm seeing a couple of issues with the conversation process.

1. Tags are not appearing in the Obsidian notes.
2. Properties are not being created correctly within the document header section.
 
I appreciate that the above might be due to my inexperience with Obsidian and MD, but I'd be grateful for any help you could offer.

I'm using Obisidian Version 1.4.3 (Installer 1.3.5). I've pasted the template used below:

---

{created-at-block}_Created at {created-at}._{end-created-at-block}
{updated-at-block}_Last updated at {updated-at}._{end-updated-at-block}
{source-url-block}_Source URL []({source-url})._{end-source-url-block}

{tags-block}
Tagged 
```
{tags}
```
{end-tags-block}

---

{title-block}# {title}{end-title-block}

{content-block}{content}{end-content-block}

**Comment by akosbalasko** [comment, marked as answer]
Hi @CoachKidd ,

I think the problem is related to the template. In order to make the properties work in Obsidian, the template's frontmatter should be produced to fit with a valid Yaml, like this: 
```
--- 
{tags-block} tags: {tags}{end-tags-block}
--- 
```

Please try this, I update the default template to match with the yaml props soon. 
Thank you!

**Reply by CoachKidd** [reply]
Many thanks.  I'll try this when I get home.

Are you able to explain the formatting syntax, please?

Ideally, I'd like to add properties from the Evernote variables.  What are they?

**Reply by akosbalasko** [reply]
Ok, let's assume we have the following line: 
```{tags-block} tags: {tags}{end-tags-block}```
```{tags-block}``` and ```{end-tags-block}``` are placeholders for Yarle to recognize the place which should be filled in with tags (or with other variables in other cases). 
What is between them is going to be generated, but {tags} are going to be replaced by the list of the tags you have in the evernote note.  ":" Separates the property name from its value in Yaml, so its left value is going to be used as property name, and what is after it will be the value of the property in Obsidian. 
The same syntax is followed in any other cases, those properties are supported that are shown in the template, namely these: 
```
{created-at-block}Created at: {created-at}.{end-created-at-block}
{updated-at-block}Last updated at: {updated-at}.{end-updated-at-block}
{source-url-block}Source URL: {end-source-url-block}
{tags-block}Tags: {tags}{end-tags-block}
{title-block}Title: {title}{end-title-block}
```
and for completeness the content, but I think it should not be added to the frontmatter :) 
{content-block}{content}{end-content-block}

**Comment by akosbalasko** [comment]
Hi @CoachKidd ,
I released a new version (5.3.2) in which the default template should produce a frontmatter that works with Properties in Obsidian.

**Reply by CoachKidd** [reply]
Thank you so much!
